### DISCUSSION TITLE
Use Video Files For Cutscenes

### DISCUSSION THREAD

**Discussion by Meowthkid7179** [KadeArchive/Kade-Engine#956] [Ideas]
Can we make it so we can use MP4 or other files formats for cutscenes in the Kade engine

**Comment by ghost** [comment]
From the looks of it they're using the code from [this](https://github.com/GrowtopiaFli/openfl-haxeflixel-video-code)
There is also tutorials on YouTube on how to use it by BrightFyre and Bbpanzu.

The only thing I could say about how Week 7 does videos is that Week 7's .js is practically just the source code but simplified. The only thing I know from the .js is that the code for videos is called "FlxVideo" but that's all I know unfortunately. Would be nice if someone was able to decompile the .js.

**Reply by IceyMeowth123** [reply]
im on my other account but thanks
